components/Send/components/Confirm.tsx

```diff
@@ -43,7 +43,7 @@ const Confirm: React.FunctionComponent<ConfirmProps> = ({ closeModal, confirmSen
         noSyncingStatus={true}
         noDrawMenu={true}
       />
-      <ScrollView contentContainerStyle={{ display: 'flex', justifyContent: 'flex-start' }}>
+      <ScrollView contentContainerStyle={{ display: 'flex', justifyContent: 'flex-start' }} showsVerticalScrollIndicator={true} persistentScrollbar={true} indicatorStyle={'white'} testID="send.scrollView">
```

Comment:
A couple of things...
* Even with those props I cannot see the scrollbar in my Android Device, either in my emulator, I don't know what is happening.
* if there is a decision about change this behavior in this screen, I think we need to replicate it in all the screens that they have a `ScrollView` component... just for coherence.

Reply:
2) hmm, good point